test/integration/flags.go

```diff
@@ -33,7 +33,7 @@ func TestMain(m *testing.M) {
 
 var binaryPath = flag.String("binary", fmt.Sprintf("../../out/%s-amd64/minishift", runtime.GOOS), "path to minishift binary")
 
-var vmDriver = getenv("VM_DRIVER", "virtualbox") // Get the driver value from environment variable or use default
+var vmDriver = getenv("MINISHIFT_VM_DRIVER", "virtualbox") // Get the driver value from environment variable or use default
```

Comment:
Right, we don't need the _vmDriver_ var. That is what the whole issue is about. I have a feeling that we have some misunderstanding what we want to achieve here. Also this needs to be synced with ull request #433. I recommend we close this pull request and first get #433 merged and take it from there.